src/makers/Milesight/index.ts

```diff
@@ -0,0 +1,118 @@
+import bobcat from './bobcat'
```

Comment:
This file belongs in `src/makers/index.ts`

`src/makers/Milesight/index.ts` Should look something like...
```ts
import antennas from './antennas'
import hotspots from './hotspots'

export default {
  antennas,
  hotspots,
  id: 9999, // your maker id
  supportEmail: 'support@milesight.asdf.com', // your support email
}
```